### ISSUE TITLE
Periodic Note wont open

### ISSUE THREAD

**Issue by JobeEnterprise** [mirnovov/obsidian-homepage#51]
### Details

I'm no expert, excuse me if I'm missing something obvious.
I want my daily periodic note to open on startup. I get the "Homepage 'the name of the note' does not exist". In periodic notes settings, the name of the file is YYYY-MM-DD-[W]W

### Steps to reproduce

In periodic notes settings, the name of the daily note file is YYYY-MM-DD-[W]W
In homepage settings, I have tried "YYYY-MM-DD-[W]W" and moment().format("YYYY-MM-DD-[W]W");

### Debug information

```shell
{"version":3,"homepages":{"Main Homepage":{"value":"[[YYYY-MM-DD-[W]W]]","kind":"Date-dependent file","openOnStartup":true,"hasRibbonIcon":true,"openMode":"Keep open notes","manualOpenMode":"Keep open notes","view":"Default view","revertView":true,"refreshDataview":false,"autoCreate":false,"autoScroll":false,"pin":false,"commands":[]}},"separateMobile":false,"_defaultViewMode":"default","_livePreview":"default","_focusNewTab":"default","_plugins":["obsidian-admonition","code-block-copy","obsidian-kanban","periodic-notes","templater-obsidian","dataview","obsidian-omnivore","homepage","obsidian-projects","obsidian-tasks-plugin"],"_internalPlugins":["file-explorer","global-search","switcher","graph","backlink","outgoing-link","tag-pane","page-preview","templates","note-composer","command-palette","editor-status","bookmarks","outline","word-count","file-recovery"],"_obsidianVersion":"1.2.8"}
```

### Checklist

- [X] I updated to the latest version of the plugin.

**Comment by mirnovov** [comment]
It appears what's going on is that some of the daily notes don't exist when Homepage runs on startup. It'll only create them for you if "Auto-create" is set to true, but right now it appears that you have it turned off.

 If you turn it on, then things should work as you expect. If this still doesn't fix your problem and I've got it wrong then Iet me know and I'll reopen this.

**Comment by JobeEnterprise** [comment]
Thank you for replying. Using "Auto-create" creates the note without a template. The temporary fix was to add the command "Periodic Notes: Open Daily Note". However, this still gives the note does not exist error.

**Comment by mirnovov** [comment]
This is still intended behaviour, since the "Date-dependent file" option is designed to *be compatible with* Periodic Notes but not necessarily support everything with it. However, this seems like a clear gap in the featureset so I'll add proper Periodic Notes support as a feature.

**Comment by mirnovov** [comment]
Should be fixed with 4bd6bcd. Might take a while to get merged though, it's a fairly complex change with a lot of side effects.
